docs/tools/flow-cli/flix.md

```diff
@@ -1,35 +1,124 @@
+description: Flow Interaction Templates (FLIX) on Flow from the command line
 sidebar_position: 12
 ---
+## Introduction
 
-The Flow CLI provides a command to execute Flow Interaction Templates (FLIX). FLIX are an easy way to interact with Flow. They are a standard for distributing Cadence scripts and transactions, and metadata about them, in a way that is consumable by tooling and wallets. FLIX are audited for correctness and safety by an ecosystem of auditors.
+The Flow CLI provides a `flix` command with a few sub commands `execute` and `package`. Get familiar with Flow Interaction Templates [(FLIX)](https://github.com/onflow/flips/blob/main/application/20220503-interaction-templates.md). FLIX are a standard for distributing Cadence scripts and transactions, and metadata in a way that is consumable by tooling and wallets. FLIX can be audited for correctness and safety by auditors in the ecosystem.
 
 ```shell
-flow flix <query> [<argument> <argument>...] [flags]
+>flow flix
+execute, package
+
+Usage:
+  flow flix [command]
+
+Available Commands:
+  execute     execute FLIX template with a given id, name, or local filename
+  package     package file for FLIX template fcl-js is default
+
+
 ```
 
-Queries can be a FLIX id, name, or path to a local FLIX file.
+### Execute
 
-## Example Usage
+The Flow CLI provides a `flix` command to `execute` FLIX. The Cadence being execute in the FLIX can be a transaciton or script.
+
+```shell
+flow flix execute <query> [<argument> <argument>...] [flags]
+```
+
+Queries can be a FLIX `id`, `name`, or `path` to a local FLIX file.
+
+### Execute Usage
 
 ```shell
 # Execute a FLIX transaction by name on Testnet
-flow flix transfer-flow 5.0 "0x123" --network testnet --signer "testnet-account"
+flow flix execute transfer-flow 5.0 "0x123" --network testnet --signer "testnet-account"
 ```
 
 ```shell
 # Execute a FLIX script by id on Testnet
-flow flix bd10ab0bf472e6b58ecc0398e9b3d1bd58a4205f14a7099c52c0640d9589295f --network testnet
+flow flix execute bd10ab0bf472e6b58ecc0398e9b3d1bd58a4205f14a7099c52c0640d9589295f --network testnet
 ```
 
 ```shell
 # Execute a local FLIX script by path on Testnet
-flow flix ./multiply.template.json 2 3 --network testnet
+flow flix execute ./multiply.template.json 2 3 --network testnet
+```
+
+The Flow CLI provides a `flix` command to `package` up generated plain and simple JavaScript. This JavaScript uses FCL (Flow Client Library) to call the cadence the Flow Interaction Templates (FLIX) is based on. 
+
+<Callout type="info">
+Currently, flix `package` command only supports generating FCL (Flow Client Library) specific JavaScript, there are plans to support other languages like golang.
+</Callout>
+
+
+```shell
+flow flix package <query> [flags]
+```
+
+## Package
+
+Queries can be a FLIX `id`, `name`, or `path` to a local FLIX file. This command leverages [@onflow/fcl](https://www.npmjs.com/package/@onflow/fcl) to ingest a template that has Cadence code. 
```

Comment:
Would link from docs to other docs...this link: https://developers.flow.com/tools/clients/fcl-js

Reply:
I was thinking dev would use the npm package to get the fcl in order to use it.

Reply:
I can change to point to FCL in the docs.